src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -1910,6 +1868,7 @@ MavlinkReceiver::handle_message_rc_channels_override(mavlink_message_t *msg)
 	}
 
 	// publish uORB message
+	rc.timestamp = hrt_absolute_time();
```

Comment:
Only nees one hrt call since it's anyways the same.
```suggestion
	rc.timestamp = rc.timestamp_last_signal = hrt_absolute_time();
```